packages/commonwealth/client/scripts/helpers/ContractHelpers/Contest.ts

```diff
@@ -0,0 +1,125 @@
+import { AppError } from '@hicommonwealth/core';
+import { ContestAbi } from './Abi/ContestAbi';
+import ContractBase from './ContractBase';
+import NamespaceFactory from './NamespaceFactory';
+
+class Contest extends ContractBase {
+  namespaceFactoryAddress: string;
+  namespaceFactory: NamespaceFactory;
+
+  constructor(contractAddress: string, factoryAddress: string, rpc: string) {
+    super(contractAddress, ContestAbi, rpc);
+    this.namespaceFactoryAddress = factoryAddress;
+  }
+
+  async initialize(withWallet: boolean = false): Promise<void> {
+    await super.initialize(withWallet);
+    this.namespaceFactory = new NamespaceFactory(
+      this.namespaceFactoryAddress,
+      this.rpc,
+    );
+    await this.namespaceFactory.initialize();
+  }
+
+  /**
+   * deploys a new recurring contest and configures the feeManager to divert feeShare of
+   * stake fees to new contest
+   * @param namespaceName namespace name to configure for
+   * @param contestInterval the recurrng contest interval in ms
+   * @param winnerShares the percent of each respective winning places prize(ie 5 = 5%)
+   * @param stakeId the id of the community stake token, defaults to 2
+   * @param prizeShare the % amount of the prize pool to payout each interval(ie 5 = 5%)
+   * @param voterShare the % amount of prize pool claimable by voters each week(ie 5 = 5%)
+   * @param feeShare the % amount of community stake fees to divert to contest(ie 5 = 5%)
```

Comment:
I can't see anywhere on the designs that admin is able to set `voterShare` and `feeShare` during process of creating a contest. How do I get this values from?

Reply:
We need to ask product about defaults for these. Not seeing them on the UI designs

Reply:
From slack:

Thanks got it for FeeShare.
But I think need to clarify what voterShare is.
Its the % of the totalPrize that the voters of winning content receive, not the winner.
So if the total prize is 1 ETH a 20% voter share implies .20 ETH is shared by voters on the winning content

Reply:
This was determined as feeShare will always = 100 and voterShare will always equal 20